main/schemas/base_class.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+
+
+class BaseClass(BaseModel):
```

Comment:
This class name is not specific enough. Please rename it so it can be self-explainable